### DISCUSSION TITLE
BPMN Workflow Quickstart

### DISCUSSION THREAD

**Discussion by teshdor** [viewflow/viewflow#378] [General]
I've  followed all the steps get the workflow section working, but after migrations I get the below error when trying to access the page:

Exception Value: | no such table: viewflow_base_task

I'm using the sqlite database and I see the migration for base_process but not for base_task.

What am I doing wrong?

EDIT: I figured it out, I had to run: python manage.py makemigrations viewflow_base
This is not located anywhere. Also the viewflow guide is riddled with code mistakes. Very frustrating.

EDIT2: The non pro version of 2.0 does not work with the new documentation. It's a mess.

**Comment by kmmbvnr** [comment]
Probably this could happen b/c missing a step to add `viewflow.workflow` to INSTALLED_APP in the tutorial

I've released viewflow 2.0.0b3 and fix the tutorial https://docs-next.viewflow.io/workflow/quick_start.html
